libsoftwarecontainer/src/gateway/environment/envgatewayparser.cpp

```diff
@@ -21,23 +21,23 @@
 
 namespace softwarecontainer {
 
-ReturnCode EnvironmentGatewayParser::parseEnvironmentGatewayConfigElement(
+bool EnvironmentGatewayParser::parseEnvironmentGatewayConfigElement(
     const json_t *element,
     EnvironmentVariable &result,
     const EnvironmentVariables &store)
 {
 
-    if (isError(requireNonEmptyKeyValue(element, "name", result.first))
-     || isError(requireNonEmptyKeyValue(element, "value", result.second))) {
-        return ReturnCode::FAILURE;
+    if (!requireNonEmptyKeyValue(element, "name", result.first)
+     || !requireNonEmptyKeyValue(element, "value", result.second)) {
```

Comment:
This indentation is actually not according to the guidelines, the '||' should be aligned with the start of the first parameter.